content/tutorials/quickstart/py-create-accounts-send-xrp.md

```diff
@@ -0,0 +1,437 @@
+2. Retrieve the accounts from seed values.
+3. Transfer XRP between accounts.
+
+When you create an account, you receive a public/private key pair offline. Your account does not appear on the ledger until it is funded with XRP. This example shows how to create accounts for Testnet, but not how to create an account that you can use on Mainnet.
+
+[![Token Test Harness](img/quickstart-py2.png)](img/quickstart-py2.png)
+
+## Prerequisites
+
+To get started, create a new folder on your local disk and install the Python library using `pip`.
+
+```
+    pip3 install xrpl-py
+```
+
+Download and expand the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip){.github-code-download} archive.
+
+**Note:** Without the Quickstart Samples, you will not be able to try the examples that follow. 
+
+## Usage
+
+To get test accounts:
+
+1. Open and launch `lesson1-send-xrp.py`.
+2. Click **Get Standby Account**.
+3. Click **Get Operational Account**.
+4. Click **Get Standby Account Info**.
+5. Click **Get Operational Account Info**.
+5. Copy and paste the **Standby Seed** and **Operational Seed** fields to a persistent location, such as a Notepad, so that you can reuse the accounts after reloading the form.
+
+[![Standby and Operational Accounts](img/quickstart-py3.png)](img/quickstart-py3.png)
+
+You can transfer XRP between your new accounts. Each account has its own fields and buttons.
+
+To transfer XRP from the Standby account to the Operational account:
+
+1. On the Standby (left) side of the form, enter the **Amount** of XRP to send.
+2. Copy and paste the **Operational Account** field to the Standby **Destination** field.
+3. Click **Send XRP>** to transfer XRP from the standby account to the operational account
+
+[![Transferred XRP](img/quickstart-py4.png)](img/quickstart-py4.png)
+
+To transfer XRP from the Operational account to the Standby account:
+
+1. On the Operational (right) side of the form, enter the **Amount** of XRP to send.
+2. Copy and paste the **Standby Account** field to the Operational **Destination** field.
+3. Click **&lt;Send XRP** to transfer XRP from the Operational account to the Standby account.
+
+# Code Walkthrough
+
+You can download the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip){.github-code-download} in the source repository for this website.
+
+## mod1.py
+
+The mod1.py module contains the business logic for interacting with the XRP Ledger.
+
+Import the XRPL and JSON libraries.
+
+```python
+import xrpl
+import json
+```
+
+### get_account
+
+This method lets you get an existing account by providing a seed value. If you provide no seed value, the method creates a new account for you.
+
+Import required methods.
+
+```python
+def get_account(_seed):
+    from xrpl.clients import JsonRpcClient
+    from xrpl.wallet import Wallet
```

Comment:
I think it's better practice to place any imports at the top of the file, plus we avoid repetition.